### DISCUSSION TITLE
Jump back after "goto definition"

### DISCUSSION THREAD

**Discussion by EricCrosson** [helix-editor/helix#1552] [Q&A] [answered]
Howdy,

is there a way to "jump back" after `g d` (Goto definition)?

In emacs I enjoy the [tags](https://www.emacswiki.org/emacs/EmacsTags) system that tracks jumps with a stack. This lets me jump to a definition, move around the buffer and root around for a while, and then pop back. I can follow a trail of dereferences really deep, but always find my way back to my starting point by mashing that "pop back" button enough times.

I've explored `g p` (Goto previous buffer), but I'm not always working in a different buffer. `g .` (Goto last modification) doesn't quite fit because often I'm just investigating without editing.

It's quite possible I've missed something blatantly obvious, in which case thanks in advance for pointing me in the right direction!
Cheers

**Comment by hovsater** [comment, marked as answer]
`Ctrl-o` to jump backwards in the jumplist. `Ctrl-i` to move forward. See https://docs.helix-editor.com/keymap.html for more information.

**Reply by EricCrosson** [reply]
That'll do it! Beautiful. Thank you @kevinsjoberg
